src/components/CreateModal/CreateModalChooser.js

```diff
@@ -11,14 +11,16 @@ const postTypes = Object.keys(POST_TYPES)
 export default class CreateModalChooser extends Component {
   render () {
     const { location } = this.props
+    const pathname = location.pathname.slice(-1) === '/' ? location.pathname : `${location.pathname}/`
+    const params = location.search ? `&${location.search.substring(1)}` : ''
 
     return <div styleName='chooser'>
-      <h1>What would you like to create?</h1>
+      <h1>{params.includes('lat') && params.includes('lng') ? 'New Post at this location: ' : ''}What would you like to create?</h1>
```

Comment:
For clarity maybe make this `const hasLocation = params.includes(`lat`) && ...` above the `return` and then use that in the render...